<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;
background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RedButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
                        border: 10px solid rgb(239, 22, 20);
					 border-radius: 50px;
                        color:rgb(239, 22, 20);
                        background-color:white;}
QPushButton:pressed {
                        color:rgb(239, 22, 20);
                         background-color:rgb(239, 22, 20); }&quot;
</string>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BlueButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
                        border: 10px solid rgb(19, 152, 236);
					 border-radius: 50px;
                        color:rgb(19, 152, 236);
                        background-color:white;}
QPushButton:pressed {
                        color:rgb(19, 152, 236);
                         background-color:rgb(19, 152, 236); }&quot;
</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>170</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
					 border-radius: 15px;
                        color:black;
                        background-color:rgb(63, 207, 69);}

</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid black;
border-radius: 10px;
background: white;
padding: 4px;
text-align: right;
margin-right: 6ex;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="TurnLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simon Says</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="YellowButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
                        border: 10px solid rgb(253, 249, 40);
					 border-radius: 50px;
                        color:rgb(253, 249, 40);
                        background-color:white;}
QPushButton:pressed {
                        color:rgb(253, 249, 40);
                         background-color:rgb(253, 249, 40); }&quot;
</string>
    </property>
    <property name="text">
     <string>Yellow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GreenButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ 
                        border: 10px solid rgb(66, 211, 72);
					 border-radius: 50px;
                        color:rgb(66, 211, 72);
                        background-color:white;}
QPushButton:pressed {
                        color:rgb(66, 211, 72);
                         background-color:rgb(66, 211, 72); }&quot;
</string>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eurostile</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>SIMON</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>RedButton</zorder>
   <zorder>BlueButton</zorder>
   <zorder>StartButton</zorder>
   <zorder>TurnLabel</zorder>
   <zorder>YellowButton</zorder>
   <zorder>GreenButton</zorder>
   <zorder>ProgressBar</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
